src/runtime/profiler.cpp

```diff
@@ -461,6 +475,17 @@ void profiler::install(service_spec &)
                     COUNTER_TYPE_NUMBER_PERCENTILES,
                     "");
             }
+            if (dsn_config_get_value_bool(
```

Comment:
Why didn't you use flags to define `rpc_request_dropped_before_execution_when_timeout`?